tests/integration_tests/db_engine_specs/trino_tests.py

```diff
@@ -87,3 +89,71 @@ def test_get_extra_params_with_server_cert(self, create_ssl_cert_file_func: Mock
         self.assertEqual(connect_args.get("http_scheme"), "https")
         self.assertEqual(connect_args.get("verify"), "/path/to/tls.crt")
         create_ssl_cert_file_func.assert_called_once_with(database.server_cert)
+
+    @patch("trino.auth.BasicAuthentication")
+    def test_auth_basic(self, auth: Mock):
+        database = Mock()
+
+        auth_params = dict(username="username", password="password",)
```

Comment:
nit, I believe we prevalently use this notation in the codebase:
```suggestion
        auth_params = {"username": "username", "password": "password"}
```